lib/pacio_test_kit/interactions_test.rb

```diff
@@ -1,6 +1,36 @@
 module PacioTestKit
   module InteractionsTest
     # TODO: All helper methods for interactions tests (CRUD) will be added here.
+    def create_and_validate_resources(_resource, _tag = '')
+      # make FHIR resource object
+      new_resource = new Object.const_get(resource_type)
+
+      # send object to create on server
+      fhir_create(new_resource)
+
+      # validate the resource from the request.response
+      status = request.response[:status]
+      headers = request.response[:headers]
+
+      # create SHALL return status and headers
+      validate_status('create', status, req_num)
```

Comment:
there will not be any `req_num` to pass here since we are only creating one resource. should update the `validate_status` to have that param as optional